src/BaseBuilder.js

```diff
@@ -65,6 +65,20 @@ export default class BaseBuilder {
   }
 
   /**
+   * Set what property should be automatically created.
+   *
+   * Supported Options:
+   * - placeholders
+   * - labels
```

Comment:
Should `- none` also be a supported option?

https://github.com/gcanti/tcomb-form/blob/master/GUIDE.md#automatically-generated-placeholders

> Or auto: 'none' if you don't want neither labels nor placeholders

Reply:
yeah, good call